spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRemoveOrphanFilesProcedure.java

```diff
@@ -647,19 +648,33 @@ public void testRemoveOrphanFilesProcedureWithPrefixMode()
             "CALL %s.system.remove_orphan_files("
                 + "table => '%s',"
                 + "equal_schemes => map('file1', 'file'),"
+                + "equal_authorities => map('localhost', '%s'),"
```

Comment:
rather than modifying an existing test I think we should just have a simple & separate test method that reproduces the underlying issue